src/types/actors/ActorReentrancyOptions.ts

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * ActorReentrancyOptions provides options for configuring reentrancy in actors.
+ * For more, see https://docs.dapr.io/developing-applications/building-blocks/actors/actor-reentrancy/.
+ */
+export type ActorReentrancyOptions = {
```

Comment:
Should we have ActorReentrancyOptions type present in ActorRuntimeOptions.ts itself? As `ReentrancyOptions` will be mainly used by `RuntimeOptions`.

Reply:
Are you talking about moving the type to that file, or embedding these properties directly in `ActorRuntimeOptions`? If the former, it's just a cosmetic change and maybe keeping in a separate file is cleaner. For the latter, it aligns to the JSON model required by runtime.

Reply:
Yeah, the former. To me it seemed no need of an extra file and have this type reside in `RuntimeOptions` file. But, I am fine with keeping it separate as well.